ui/src/app/pages/admin/admin-institution-edit.spec.tsx

```diff
@@ -514,64 +467,151 @@ describe('AdminInstitutionEditSpec - add mode', () => {
     expect(wrapper).toBeTruthy();
   });
 
+  it('should throw error for a new Institution if the display name is more than 80 characters', async() => {
```

Comment:
this new test failed (why I didn't include it in the other PR)

but the fix was easy: see below.